### PR TITLE
[fix] Server-side fetch request should have headers (#969)

### PR DESCRIPTION
**Pull request by aolose**
Server-side fetch  in load function will include original request headers.

I'm only test it in the Page file , if server-side fetch not in load , headers will lost :(  ,   

### Please don't delete this checklist! Before submitting the PR, please make sure you do the following:
- [x] It's really useful if your PR references an issue where it is discussed ahead of time. In many cases, features are absent for a reason. For large changes, please create an RFC: https://github.com/sveltejs/rfcs
- [x] This message body should clearly illustrate what problems it solves.
- [ ] Ideally, include a test that fails without this PR but passes with it.

### Tests
- [ ] Run the tests with `pnpm test` and lint the project with `pnpm lint` and `pnpm check`

### Changesets
- [ ] If your PR makes a change that should be noted in one or more packages' changelogs, generate a changeset by running `pnpx changeset` and following the prompts. All changesets should be `patch` until SvelteKit 1.0

### CODE REVIEW

**Comment by benmccann** [comment]
please fix the failing lint job

File: packages/kit/src/runtime/server/page/load_node.js [hunk 1]
```diff
@@ -101,6 +101,12 @@ export async function load_node({
 					};
 				}
 
+				//merge  headers  from  request
```
Comment (benmccann) [inline packages/kit/src/runtime/server/page/load_node.js:104]: we usually put a space after `//`
```suggestion
				// merge  headers  from  request
```

Comment (bluwy) [reply packages/kit/src/runtime/server/page/load_node.js:105]: There are two spaces between words too, we should have one space only.

**Comment by benmccann** [comment]
@aolose this PR would need to be rebased and my comment would need to be addressed

**Comment by aolose** [comment]
I deleted the repo I forked earlier, so I fork it again.
The new pr is here: #3631.

**Pull request closed without merging**